### ISSUE TITLE
is there error  lines204-214  in EfficientDet-model.py ???

### ISSUE THREAD

**Issue by y78h11b09** [zylo117/Yet-Another-EfficientDet-Pytorch#323]
I am very confusion for you comments  at line 207 and 213, I think they shoud be " Connections for P5_0 and P6_1 to P5_1 respectively" and "Connections for P4_0 and P5_1 to P4_1 respectively  ", are they right ?

   # Weights for P5_0 and P6_1 to P5_1
        p5_w1 = self.p5_w1_relu(self.p5_w1)
        weight = p5_w1 / (torch.sum(p5_w1, dim=0) + self.epsilon)
        # Connections for P5_0 and P6_0 to P5_1 respectively
        p5_up = self.conv5_up(self.swish(weight[0] * p5_in + weight[1] * self.p5_upsample(p6_up)))

        # Weights for P4_0 and P5_1 to P4_1
        p4_w1 = self.p4_w1_relu(self.p4_w1)
        weight = p4_w1 / (torch.sum(p4_w1, dim=0) + self.epsilon)
        # Connections for P4_0 and P5_0 to P4_1 respectively  
        p4_up = self.conv4_up(self.swish(weight[0] * p4_in + weight[1] * self.p4_upsample(p5_up)))

**Comment by zylo117** [comment]
yes, you're right, it's fixed now. Thanks!
